app/src/main/contrib/components/Responsibilities/Coverage/CoverageSetComponent.tsx

```diff
@@ -14,7 +14,7 @@ export class CoverageSetComponent extends React.Component<CoverageSetComponentPr
             <td>{ this.props.set.name }</td>
             <td>{ this.props.set.vaccine }</td>
             <td>{ this.props.set.activity_type }</td>
-            <td>{ this.props.set.gavi_support_level }</td>
+            <td>{ this.props.set.gavi_support }</td>
         </tr>;
     }
 }
```

Comment:
This was the first time we had regenerated the models since this change was made. It didn't seem worth making a separate PR for it, but I can if you like.